### ISSUE TITLE
Left-hand Column of the Dashboard Needs ARIA Landmark

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#3087]
### Descriptive summary

In Hyrax 2.1.0.rc1, the left-hand column of the dashboard needs an ARIA landmark to assist screen reader users in understanding and navigating the structure of the page.

### Rationale

The left-hand column of the dashboard contains major navigation for the admin section of the site and wrapping the list of links in a `<nav>` element will assist screen reader users with navigating major sections of the dashboard,

### Expected behavior

Expect major regions like navigation to have landmark designations like `<nav>` .

### Actual behavior

Missing `<nav>` element

Nurax Version
ENV: production 
SHA: 4f5a5817690ed131aa4aab72b993526225ab794c 
Branch: master 

### Related work

Violates [WCAG 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html#content-structure-separation-programmatic-intent-head) guidelines

**Comment by adamjarling** [comment]
Duplicated by #3085
